global-pixel-definitions/ignore_params.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Convention nit: all other dirs and paths in this repo are using `_`, this one uses `-` in the directory, then `_` in the file name.

Reply:
@sammacbeth I think convention is actually '_' for [JSON files](https://github.com/duckduckgo/pixel-schema/tree/main/schemas), and product repos, see also [this code here](https://github.com/duckduckgo/pixel-schema/blob/c5eb035335c86e701f342dc366fef0265f583931/src/file_utils.mjs#L72).

For dir naming, I was probably just looking at the Ext naming at the moment but 👍 on changing it.